WORKSPACE

```diff
@@ -46,19 +46,37 @@ load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
 
 closure_repositories()
 
-http_archive(
-    name = "org_tensorflow",
-    sha256 = "8028d51b4a911adeb9b8afa0ba6bcb99fa00a4949881cdad3ee67a8f33c8979a",
-    strip_prefix = "tensorflow-3128b43eb0bf37ac3c49cb22a6e1789d8ea346e8",
-    urls = [
-        "https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/3128b43eb0bf37ac3c49cb22a6e1789d8ea346e8.tar.gz",  # 2018-04-16
-        "https://github.com/tensorflow/tensorflow/archive/3128b43eb0bf37ac3c49cb22a6e1789d8ea346e8.tar.gz",
-    ],
+
+load("//build_rules:repo.bzl", "tensorflow_http_archive")
+
+tensorflow_http_archive(
+  name = "org_tensorflow",
+  sha256 = "cbd96914936ce3aacc39e02c2efb711f937f8ebcda888c349eab075185d7c914",
+  git_commit = "d8fac4cb80eb0c42d2550bcb720a80d29fc5f22d",
 )
 
-load("@org_tensorflow//tensorflow:workspace.bzl", "tf_workspace")
+#http_archive(
```

Comment:
remove the lines that are commented out.

Reply:
Removed.